sunpy/map/header_helper.py

```diff
@@ -133,15 +133,16 @@ def make_fitswcs_header(data, coordinate,
     meta_wcs.update(meta_instrument)
 
     if reference_pixel is None:
-        reference_pixel = u.Quantity([(shape[1] + 1)/2.*u.pixel, (shape[0] + 1)/2.*u.pixel])
+        reference_pixel = u.Quantity([(shape[1] - 1)/2.*u.pixel, (shape[0] - 1)/2.*u.pixel])
     if scale is None:
         scale = [1., 1.] * (u.arcsec/u.pixel)
 
     meta_wcs['crval1'], meta_wcs['crval2'] = (coordinate.spherical.lon.to_value(meta_wcs['cunit1']),
                                               coordinate.spherical.lat.to_value(meta_wcs['cunit2']))
 
-    meta_wcs['crpix1'], meta_wcs['crpix2'] = (reference_pixel[0].to_value(u.pixel),
-                                              reference_pixel[1].to_value(u.pixel))
+    # Add 1 to go from input 0-based indexing to FITS 1-based indexing
+    meta_wcs['crpix1'], meta_wcs['crpix2'] = (reference_pixel[0].to_value(u.pixel) + 1,
```

Comment:
but what if the reference pixel given is already in FITS based indexing - i.e. you pull it from the header of another map 🤔 

but maybe this is a silly question ...

Reply:
We will probably have to be more explicit in docs

Reply:
The docs already explicitly say that the reference pixel is zero indexed. Which was a lie :see_no_evil: 

> this argument is zero indexed (Python convention) not 1 indexed (FITS convention)

Reply:
Ideally one could do `m.reference_pixel` to get a zero index reference pixel, but that just returns the one-based index pixels at the moment... if we're going to make everything zero indexed then I guess we should change the `reference_pixel` property too?